src/modules/EditTasks.js

```diff
@@ -0,0 +1,43 @@
+import UpdateIndex from './UpdateIndex.js';
+import { RemoveFromLocal } from './RemoveTask.js';
+import { GetTasks } from './ReadTasks.js';
+import StoreTask from './UpdateTasks.js';
+
+const EditTask = (idTask) => {
+  const idLi = document.getElementById(idTask);
+  const addedInputTask = idLi.querySelector('input[type=text]');
+  const iconContainer = document.getElementById(`icon-container-${idTask}`);
+  idLi.classList.add('edit-task');
+  addedInputTask.classList.add('edit-task');
+  addedInputTask.disabled = false;
+  addedInputTask.focus();
+  iconContainer.innerHTML = `<i id="remove-task-${idTask}" class="remove-icon fa-solid fa-trash-can"></i>`;
```

Comment:
- [OPTIONAL] I noticed that after editing a task and reload the page, the modified text goes back to it initial state. Kindly look into that and fix. :point_down: 

[edit-funct.webm](https://user-images.githubusercontent.com/26597432/213548389-67f2df42-b5ac-4aa2-8ca6-3affdad6a4d4.webm)